src/engine/bank02.asm

```diff
@@ -1,17 +1,1775 @@
-Func_8000: ; 8000 (2:4000)
-	INCROM $8000, $8211
+_DuelCheckInterface: ; 8000 (2:4000)
```

Comment:
I think that, for consistency with the naming of other subroutines in ``DuelMenu_*`` submenus, I would rename this to ``OpenDuelCheckMenu`` (other labels e.g. ``OpenPlayerHandScreen`` don't have the word "Duel" in the middle but I think it would be too ambiguous here if it didn't). Same for ``DuelCheckInterface`` in home.asm. (mass replacing _DuelCheckInterface_ should do the trick)